packages/desktop-client/src/style/themes/dark.ts

```diff
@@ -176,11 +176,11 @@ export const checkboxBorderSelected = colorPalette.purple300;
 export const checkboxShadowSelected = colorPalette.purple500;
 export const checkboxToggleBackground = colorPalette.gray700;
 
-export const pillBackground = colorPalette.navy800;
+export const pillBackground = colorPalette.navy600;
```

Comment:
This change about 4 other pages as well. It also doesn't pass AA.

Reply:
Nice catch! I forgot to revert those back. I tried reusing pill colors initially but decided to use different theme variables.

I renamed some variables because it was a bit hard to follow when we interchange filters with conditions throughout the codebase. In my mind, they are 2 different things/concepts - filters = saved filters (which uses conditions), conditions = rule conditions (but also used by filters). Just thought it would be more consistent to rename some of these variables.

Reply:
Yea, I like the changes. I agree with you sentiment - lots of intermixing of terms. Was just commenting that it's a bit outside the scope of the PR. No worries, though!